starfish/core/imagestack/imagestack.py

```diff
@@ -772,20 +773,23 @@ def apply(
             # create a copy of the ImageStack, call apply on that stack with in_place=True
             image_stack = deepcopy(self)
             image_stack.apply(
-                func=func,
+                func,
+                *args,
                 group_by=group_by, in_place=True, verbose=verbose, n_processes=n_processes,
                 clip_method=clip_method,
                 **kwargs
             )
             return image_stack
 
         # wrapper adds a target `data` parameter where the results from func will be stored
-        # data are clipped or scaled by chunk using preserve_float_range if clip_method != 2
+        # data are clipped or scaled by chunk using preserve_float_range if
```

Comment:
*data is*...jkjk ;)

Reply:
Lol.

To be fair, this entire sentence doesn't make a ton of sense.

Reply:
I put up https://github.com/spacetx/starfish/pull/1551 which I'm going to have @ambrosejcarr approve since he wrote these comments.